__tests__/app.test.js

```diff
@@ -152,4 +153,58 @@ describe("app", () => {
         });
     });
   });
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("POST 201: inserts a new comment to the db and returns the new comment back to the client", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "Love this article!!!",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comments.author).toBe("butter_bridge");
+          expect(body.comments.body).toBe("Love this article!!!");
```

Comment:
It would also be good to test for the other properties, which might be more broad in some cases, for example we could expect `created_at` to be any string